canonical-web-and-design/canonical.com
# Mega menu navigation

## Done
Added the drop-down mega menu navigation.
Add the second level navigation

## QA
- Click the Products navigation item
- Check matches [the design](https://app.zeplin.io/project/5d5566ee83622d9b72e8801d/screen/5da4639ca1d84f309d063e7c) and [the copy doc](https://docs.google.com/document/d/16ltouT0Yo-2RccGiE_irzrCloYaEleqFyU34fmT9OiQ/edit#) 
- Click the Partners navigation item
- Check matches [the design](https://app.zeplin.io/project/5d5566ee83622d9b72e8801d/screen/5da4639f85f67a23e56b4080) and [the copy doc](https://docs.google.com/document/d/1M2CrdIi1IWsSFNkmk5yNk_w835IXZUK395fwAgDhbsQ/edit#)
- Click the Careers navigation item
- Check matches [the design](https://app.zeplin.io/project/5d5566ee83622d9b72e8801d/screen/5da4639ff2c331081a5ee065) and [the copy doc](https://docs.google.com/document/d/1rFQhtoczicKT6Ndlz6csh2ZjD9TZU-_WEvVxFjrHwUY/edit#)
- Go to /careers and see the second level navigation matches [the design](https://app.zeplin.io/project/5d5566ee83622d9b72e8801d/screen/5d5aa41b4718c65235584b8a) and functions
- Go to /partners and see the second level navigation matches [the design](https://app.zeplin.io/project/5d5566ee83622d9b72e8801d/screen/5d935a5e476a3d024a8c703e) and functions

Fixes https://github.com/canonical-web-and-design/canonical.com/issues/54
Fixes #58
